piper/ratbagd.py

```diff
@@ -420,6 +420,9 @@ def set_default(self):
 class RatbagdButton(_RatbagdDBus):
     """Represents a ratbagd button."""
 
+    KEY_PRESS = 0
+    KEY_RELEASE = 1
```

Comment:
Should add `KEY_PRESS_RELEASE` here as well, even though Piper for now will have only key press support.